client/src/main.tsx

```diff
@@ -1,45 +1,53 @@
-import ReactDOM from 'react-dom/client';
-import { createBrowserRouter, Navigate, RouterProvider } from 'react-router-dom';
-import App from './App.tsx';
-import './index.css';
-import ErrorPage from './pages/Library.tsx';
-import Reviews from './pages/Reviews.tsx';
-import Login from './pages/Login.tsx';
-import auth from './utils/auth.ts';
-import Signup from './pages/Signup.tsx';
-import Search from './pages/Search.tsx';
+import ReactDOM from "react-dom/client";
+import {
+  createBrowserRouter,
+  Navigate,
+  RouterProvider,
+} from "react-router-dom";
+import App from "./App.tsx";
+import "./index.css";
+import ErrorPage from "./pages/Library.tsx";
+import Reviews from "./pages/Reviews.tsx";
+import Login from "./pages/Login.tsx";
+import auth from "./utils/auth.ts";
+import Signup from "./pages/Signup.tsx";
+import Search from "./pages/Search.tsx";
 
 const router = createBrowserRouter([
   {
-    path: '/',
+    path: "/",
     element: <App />, // Use Layout as the parent route
     errorElement: <ErrorPage />,
     children: [
       {
-        path: '',
-        element: <Navigate to="/login" replace /> // Redirect root to login
+        path: "/",
+        element: <Navigate to="/library" replace />, // Redirect root to login
```

Comment:
We should not make this change. 
On load of the app it should take them to login not the library. Becauase then it is bypassing them logging in.

All of these other pages that aren't login should be behind this
`auth.isLoggedIn` like below

```
      {
        path: "/reviews",
        element: auth.loggedIn() ? (
          <Reviews />
        ) : (
          <Navigate to="/login" replace />
        ), // Protect the /home route
      },
```

Reply:
You're right. Forgot to switch it back after testing it.

Good catch!